test/services/assessmentcentre/AssessmentCentreToFsbOrOfferProgressionServiceSpec.scala

```diff
@@ -38,10 +41,22 @@ class AssessmentCentreToFsbOrOfferProgressionServiceSpec extends ScalaMockUnitSp
 
         (mockFsbRepository.progressToFsb _)
           .expects(expectedApplication)
-          .returning(Future.successful(())).once
+          .returningAsync.once
+
+        (mockApplicationRepository.find(_: String))
+          .expects(expectedApplication.applicationId)
+          .returningAsync(Option(c0)).once
+
+        (mockAuthProviderClient.findByUserIds(_: Seq[String])(_: HeaderCarrier))
+          .expects(Seq(userId), hc)
+          .returningAsync(Seq(c1)).once
+
+        (mockEmailClient.sendCandidateAssessmentCompletedMovedToFsb(_: String, _: String)(_: HeaderCarrier))
+          .expects(c1.email, c1.name, hc)
+          .returningAsync.once
```

Comment:
I'm pretty sure that you can omit the .once calls and the default is that the method is only called once